PoGo.NecroBot.Logic/State/FarmState.cs

```diff
@@ -19,43 +19,43 @@ public class FarmState : IState
                 await EvolvePokemonTask.Execute(session, cancellationToken);
             }
 
-            if (session.LogicSettings.TransferDuplicatePokemon)
+            if (session.LogicSettings.UseEggIncubators)
             {
-                await TransferDuplicatePokemonTask.Execute(session, cancellationToken);
+                UseIncubatorsTask.Execute(session, cancellationToken);
             }
 
-            if (session.LogicSettings.AutomaticallyLevelUpPokemon)
+            if (session.LogicSettings.TransferDuplicatePokemon)
             {
-                await LevelUpPokemonTask.Execute(session, cancellationToken);
+                TransferDuplicatePokemonTask.Execute(session, cancellationToken);
             }
 
             if (session.LogicSettings.UseLuckyEggConstantly)
             {
-                await UseLuckyEggConstantlyTask.Execute(session, cancellationToken);
+                UseLuckyEggConstantlyTask.Execute(session, cancellationToken);
```

Comment:
Since these tasks are no longer awaited, you'd get unobserved task exceptions if something goes wrong. That seems unfortunate. Was this intended?